api/views/submission.py

```diff
@@ -639,13 +641,17 @@ def create(
         tor_url = request.data.get("tor_url")
         # allows pre-marking submissions we know won't be able to make it through OCR
         cannot_ocr = request.data.get("cannot_ocr", "False") == "True"
+        nsfw = request.data.get("nsfw", "False") == "True"  # maps to PRAW's `.over_18`
```

Comment:
Ok I see that it's commented here, it might make sense to move this to the model (or have it at both places)